ui/admin/app/templates/scopes/scope/targets/target/manage-alias.hbs

```diff
@@ -3,6 +3,11 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 
+<Breadcrumbs::Item
```

Comment:
Do we have other pages where the breadcrumbs component is outside of the layout?